pkg/providers/pricing/client/pricingapi.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 const (
-	apiVersion = "2021-10-01-preview"
+	apiVersion = "2023-01-01-preview"
```

Comment:
why 2023? Whats the latest version? Also why preview? Just following convention?

Reply:
That's the latest version